### ISSUE TITLE
bnd-resolver-maven-plugin: Add option to only include current project artifact but not Maven dependencies

### ISSUE THREAD

**Issue by kwin** [bndtools/bnd#4207]
Currently there is either the option to consider all Maven dependencies with certain scopes or to not consider at all. When it is not considered at all, even the currently built project is not part of the repository. IMHO the current artifact should always be put into the implicit repository!

**Comment by kwin** [comment]
As workaround one can use
```
<plugin>
                    <groupId>biz.aQute.bnd</groupId>
                    <artifactId>bnd-resolver-maven-plugin</artifactId>
                    <version>${bnd.version}</version>
                    <executions>
                        <execution>
                            <goals>
                                <goal>resolve</goal>
                            </goals>
                            <phase>verify</phase>
                        </execution>
                    </executions>
                    <configuration>
                        <useMavenDependencies>true</useMavenDependencies>
                        <!-- do not include any regular scopes (https://github.com/bndtools/bnd/blob/38ef85e389678370304846d6b09ffd31f9efb87d/biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/resolve/Scope.java#L13), compare with https://github.com/bndtools/bnd/issues/4207 -->
                        <scopes>distro</scopes>
                    </configuration>
                </plugin>
```

**Comment by rotty3000** [comment]
@kwin I suspect this might have been fixed by 83f867a39 . Could you double check?

**Comment by rotty3000** [comment]
I'm going to mark this as fixed by 83f867a

If however someone finds that not to be the case please reopen the issue.
